tutorial/consuming_packages/getting_started/tool_requires/src/main.c

```diff
@@ -0,0 +1,33 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+#include <zlib.h>
+
+int main(void) {
```

Comment:
Maybe it is better to keep it very simple, remove all the Zlib stuff, and leave only just the CMake stuff exclusively. Even more, this can be simplified, instead of passing CMAKE_VERSION to the C++ code, just ``message()`` the version directly in CMakeLists.txt and it will be seen in ``cmake ...`` command.

To discuss, both approaches have pros and cons.